store/mongo/migration_1_1_4.go

```diff
@@ -52,16 +55,18 @@ func (m *migration_1_1_4) Up(from migrate.Version) error {
 
 	_, err := idxUsers.CreateOne(m.ctx, uniqueEmailIndex)
 	if err != nil {
-		return err
-	}
-
-	// drop old index
-	_, err = idxUsers.DropOne(m.ctx, OldDbUniqueEmailIndexName)
-	if err != nil && err.Error() != "(IndexNotFound) index not found with name [uniqueEmail]" {
-		return err
```

Comment:
Isn't s a simple inversion of the Drop and the Create enough?

Reply:
the code would be cleaner, but the usual usage will be to migrate the database for the new tenant, and then there will be no index so there will be no need to drop the index

Reply:
Well, you will get an error if the index doesn't already exist...